mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -106,7 +106,7 @@ private enum Mode {
 
     private static Logger LOGGER = LogManager.getLogger(MCRRestAPIObjectsHelper.class);
 
-    private static SimpleDateFormat SDF_UTC = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
+    private static final SimpleDateFormat SDF_UTC = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
```

Comment:
Should be replaced with https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/time/format/DateTimeFormatter.html which is thread safe